### PR TITLE
Release/0.6.0

### PR DESCRIPTION
**Pull request by dilyand**
Changes have been reviewed in https://github.com/snowplow-incubator/snowplow-bigquery-loader/pull/118. This is just a 'Prepare for release' PR.

### CODE REVIEW

**Review by lukeindykiewicz — APPROVED** [review]

File: common/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/common/LoaderRow.scala [hunk 1]
```diff
@@ -167,9 +176,9 @@ object LoaderRow {
 
   implicit val bqModeEncoder: Encoder[Mode] = deriveEncoder[Mode]
 
-  implicit def bqTypeEncoder: Encoder[Type] = deriveEncoder[Type]
```
Comment (chuwy) [inline common/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/common/LoaderRow.scala:170]: Wondering why I wrote it this way in a first place.

**Review by chuwy — APPROVED** [review]
Looks perfect.

File: common/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/common/LoaderRow.scala (hunk 1 above)
Comment (dilyand) [reply common/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/common/LoaderRow.scala:170]: What other way would be better? Manual derivation? Full auto?

Comment (dilyand) [reply common/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/common/LoaderRow.scala:170]: Ah, nvm, saw what you mean.

**Pull request merged**